store/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonReaderBuilder.java

```diff
@@ -83,6 +83,21 @@ public CarbonReaderBuilder projection(String[] projectionColumnNames) {
     return this;
   }
 
+  /**
+   * Accepts projection list
+   *
+   * @param projectionColumnNames
+   * @return
+   */
+  public CarbonReaderBuilder projection(List<String> projectionColumnNames) {
```

Comment:
Why  create new interface in this PR?

Reply:
This was also one of the issue. we need interface that accepts List. Interfacing with other languages is difficult if we use just array interface.

Reply:
Please describe this issue in PR content， and add test cases for it。

Reply:
ok done